.changeset/many-apricots-try.md

```diff
@@ -0,0 +1,13 @@
+---
+'@vocab/phrase': minor
+'@vocab/cli': minor
+---
```

Comment:
The `@vocab/phrase` change should be moved into a separate changeset as it requires different release notes. `phrase` is an API, so release notes for the CLI aren't relevant.

Take a look at [a previous `@vocab/phrase` release](https://github.com/seek-oss/vocab/releases/tag/%40vocab%2Fphrase%401.1.0) to get an idea of what to write in the changeset.

Reply:
Thanks for the heads-up, makes sense!